contributing-guides/style-guide.md

```diff
@@ -94,6 +100,28 @@ Use backticks on the following:
 - Extensions, ex. `.dll`.
 - Commands, ex. `ls`.
 
+### Shell-specific pages
```

Comment:
Hrm, I feel like this is overly-specific and isn't a great fit for the style guide. Most pages do not document a shell, so aren't affected by this.

Reply:
Maybe this part could get its own file. There is not that much shells, but there's a lot of builtins.
Even though it's specific, might be good to have this things standardized.

Reply:
Please view [this](https://github.com/Console-Utils/tldr/blob/shell-guide/contributing-guides/tool-oriented-guides/shells.md) file.